lib/faktory/utils.ex

```diff
@@ -68,4 +68,13 @@ defmodule Faktory.Utils do
     :os.system_time(:milli_seconds)
   end
 
+  def env do
+    cond do
+      function_exported?(Mix, :env, 1) -> Mix.env
+      Application.get_env(@app_name, :env) != nil -> Application.get_env(@app_name, :env)
```

Comment:
Sorry for missing this before, but `@app_name` doesn't exist in this context. I just saw the compile warning when I was rebasing some stuff.

Reply:
Ahh, thanks! I don't really know any good way to write automated tests for this kind of stuff... :/